src/locales/en/common.json

```diff
@@ -1327,8 +1328,8 @@
     "failed": "Failed",
     "fees": "Network fees",
     "noFees": "No fee",
-    "from": "From ({{count}})",
-    "to": "To ({{count}})",
```

Comment:
Will this not break pluralization (even though it's a super weird way of using it 🤔) that might be used somewhere in the app ? I couldn't find any "common.from" key used by checking real quick, but I might be wrong, so wanted to know if you also checked 👌

Reply:
Actually this relates to https://ledgerhq.atlassian.net/browse/LL-6588 (cf. https://github.com/LedgerHQ/ledger-live-mobile/pull/1855). It ends up in my PR because I rebased it on @juan-cortes's one.

PS: this current PR might be closed since I created a new one on @juan-cortes's fork in order to get this work merged on [the general i18n PR](https://github.com/LedgerHQ/ledger-live-mobile/pull/1855)